lib/tasuku.rb

```diff
@@ -13,6 +13,7 @@ def configure
 
     def config
       @config ||= Configatron::Store.new author: :current_user
+      @config ||= Configatron::Store.new update_answers: false
       @config
     end
```

Comment:
Yeha i thought about that too. We should add a test that at least check that an answer has an author.